### PR TITLE
Update tape.d.ts for tape v4.2.2	

### PR DESCRIPTION
**Pull request by haoqunjiang**
https://github.com/substack/tape

### CODE REVIEW

File: tape/tape.d.ts [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
 // Project: https://github.com/substack/tape
-// Definitions by: Bart van der Schoor <https://github.com/Bartvds>
+// Definitions by: Bart van der Schoor <https://github.com/Bartvds>, Haoqun Jiang <https://github.com/sodatea>
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
 /// <reference path="../node/node.d.ts" />
 
 declare module 'tape' {
-	export = tape;
+	export default tape;
```
Comment (vvakame) [inline tape/tape.d.ts]: is this right?

https://github.com/substack/tape/blob/master/index.js#L19
tape does not have `exports.default = module.exports;` syntax.
`export default tape;` is not compatible `export = tape;`.

**Comment by sodatea** [comment]
Oops.
I was thinking about using this module with the `import tape from 'tape'` or `import * as tape from 'tape'` syntax, but apparently I had chosen a wrong way to achieve that.

It's fixed now.

**Comment by sodatea** [comment]
Just squashed to get a clean git history.
ping @vvakame

**Comment by vvakame** [comment]
thanks mate!

**Pull request merged**